feeder/parsers/FR.py

```diff
@@ -62,5 +62,41 @@ def fetch_production(country_code='FR', session=None):
 
     return data
 
+def fetch_price(country_code, session=None, from_date=None, to_date=None):
+    r = session or requests.session()
+    dt_now=arrow.now(tz='Europe/Paris')
+    formatted_from = from_date or dt_now.replace(days=-1).format('DD/MM/YYYY')
```

Comment:
Ok I implement the "last point only" as default behavior for FR price.
The issue with 'position' was fixed and comitted but might be I did it after merging